Main.sublime-menu

```diff
@@ -208,14 +208,10 @@
         ]
     },
     {
-        "caption": "Preferences",
```

Comment:
This is only partly true as packages usually either put them under `Tools/Packages` or `Tools`.  When I first added the menu options people were putting them under `Tools/Packages`.

![screenshot 2016-06-24 06 49 57](https://cloud.githubusercontent.com/assets/1055125/16338220/8177f238-39d8-11e6-8bec-dca2d3675bc6.png)

This was my first package, but if I had created this package today, I would never have added these menu items as I feel the command palette is a better and more useful place.  I  honestly would love to remove the menu items entirely, but I haven't because there are some people who use the menu for some reason...but I still may remove them entirely in future as I just don't find them useful as menu items.  As I will never use the menu items, I like them tucked under Packages to not clutter up my `Tools` menu.